src/hooks/useMerkleDistributor.ts

```diff
@@ -0,0 +1,6 @@
+import { useRecoilValue } from 'recoil';
+
+import { rMerkleDistributor } from 'recoilState/merkleDistributor';
+
+export const useMerkleDistributor = (epochId: number) =>
+  useRecoilValue(rMerkleDistributor(epochId));
```

Comment:
Just inline this in the recoilState/appState

recoil is global so, let's keep all those there